src/main/resources/snippets/jython/filter-map-reduce.py

```diff
@@ -0,0 +1,4 @@
+def run(data):
+    return reduce(lambda acc, next: acc + next,
+        map(lambda x: x * 2,
+            filter(lambda x: x % 2 == 0, data)), 0)
```

Comment:
Not really, it may depends on the implementation. In CPython, to quote the documentation:
“This module exports a set of functions *implemented in C* corresponding to the intrinsic operators of Python”. I don't have check how it's implemented in Jython. The `partial` call can be a bit overhead though.